### DISCUSSION TITLE
Inconsistency in the number of steps per day

### DISCUSSION THREAD

**Discussion by ffranzese** [Ergo-Tools/ActiPASS#69] [Help and Q&A (processing, quality checking or statistics)] [answered]
Dear ActiPASS Community,

The number of steps per day is different in "ActiPASS_Daily_LongFormat_MasterFile.csv" compared to the "DailyActivity.csv" file (from the "IndividualOut" folder). 
What is the difference between "NumSteps" and "Steps"?

Best, Fabio

<img width="737" alt="Steps" src="https://github.com/Ergo-Tools/ActiPASS/assets/160128322/4fa3b21f-e842-4048-95da-e2e45b3da040">

**Comment by pasan-h** [comment, marked as answer]
Hi Fabio, 
Some files inside "IndividialOut" folder such "DailyActivity.csv" are intermediate files for quick-quality checking and possible participant feedback. They do not follow ProPASS 24h construct. However "ActiPASS_Daily_LongFormat_MasterFile.csv" follow the ProPASS 24h construct. This means any behaviour (both sedentary and and upright) which happens within a sleep-interval is not counted and steps are also not counted. I think the number of steps difference we see above is because of this. 
Also there is no gurantee that some of those files will be produced in future versions of ActiPASS. Their format can also change. However the 1s epoch output file "**** - Activity_per_s.mat" will always be produced and can be used in other workflows.
Cheers! 
/Pasan
